src/main/java/ch/uzh/ifi/hase/soprafs23/service/LobbyService.java

```diff
@@ -201,4 +201,27 @@ public List<Lobby> getLobbies() {
         return this.lobbyRepository.findAll();
 
     }
+    public boolean userIsInLobby(long userId,int accessCode){
+        Lobby lobby = lobbyRepository.findByAccessCode(accessCode);
+        User user = userRepository.findById(userId);
+        if(lobby == null){
+            throw new ResponseStatusException(HttpStatus.NOT_FOUND, "Lobby with accessCode " + accessCode + " does not exist");
+        }
+        if(user == null){
+            throw new ResponseStatusException(HttpStatus.NOT_FOUND, "User with id " + userId + " does not exist");
+        }
+        return lobby.isUserInLobby(user);
+    }
+    public void deleteLobbyAndUsers(int accessCode) {
```

Comment:
Please make your code more readable. Apply decent formatting. Add spacing between methods. I think this can be done automatically with in IntelliJ Ctrl+Alt+L.